apps/dot-voting/.eslintrc.js

```diff
@@ -54,6 +54,31 @@ module.exports = {
     "react/no-unused-prop-types": "warn",
     "sort-imports": ["warn", { "ignoreDeclarationSort": true }]
   },
+  "overrides": [
+    {
+      "files": ["test/*.test.js"],
+      "env": {
+        "mocha": true,
+      },
+      "globals": {
+        "contract": "readonly",
+        "artifacts": "readonly",
+        "web3": "readonly",
+        "assert": "readonly",
+        "expect": "readonly",
+      },
+      rules: {
+        "import/no-unused-modules": [
+          "off",
+          {
+            unusedExports: true,
+            missingExports: true,
+            ignoreExports: [],
+          }
+        ],
+      }
+    }
+  ],
```

Comment:
How did `dot-voting` changes sneak into this PR?

Reply:
I'll clean anything that is not strictly related to the scope